ThrowIT/ViewControllers/TimelineViewController.m

```diff
@@ -181,7 +183,31 @@ - (NSInteger)tableView:(nonnull UITableView *)tableView numberOfRowsInSection:(N
 }
 
 -(void)tableView:(UITableView *)tableView didSelectRowAtIndexPath:(NSIndexPath *)indexPath{
```

Comment:
does adding a gesture recognizer to the table cell achieve the same behavior?

Reply:
I was not able to get it to work with a gesture recognizer; previously I had the didSelectRowAtIndexPath for the cell  to segue into a details view and a separate double tap gesture recognizer to the cell that never got called because the segue happened too quick before I could make a second tap. I added a delay to the segue action here as a solution after doing some research. Please let me know if there's a better way of going about the implementation.

Reply:
your solution makes sense. i thought we could've used gesture recognizers instead here. looks good to me!